components/list/list.html

```diff
@@ -80,6 +81,40 @@ <h3 class="font-bold font-size-6">Basic List</h3>
                         </code><button id="button" class="absolute snippet-btn"><i class="fas fa-copy"></i></button></pre>
                 </div>
             </section>
+
+            <!-- spaced list -->
+            <section class="my-7">
+                <div class="component-container flex flex-col">
+                    <div class="component-heading p-2">
+                        <h3 class="font-bold font-size-6">Spaced List</h3>
+                    </div>
+                    <div class="component-display flex justify-center flex-col p-5">
+                        <li class="list-item-container spaced-list">
+                            <span>List Item 1</span>
+                        </li>
+                        <li class="list-item-container spaced-list">
+                            <span>List Item 2</span>
+                        </li>
+                        <li class="list-item-container spaced-list">
+                            <span>List Item 3</span>
+                        </li>
+                    </div>
+                    <!-- code-snippet  -->
+                    <pre id="myInput" class="relative scroll-style"><code class="language-html">
+        &lt;li class=&quot;list-item-container spaced-list&quot;&gt;
```

Comment:
I personally used script to change '<' to '&lt'. That way your html structure will look neat.

Reply:
ok I will look into it